VERSION

```diff
@@ -1 +1 @@
-0.2.6
\ No newline at end of file
+0.2.8
```

Comment:
Sorry, I should have been more clear on that. We increase the patch version for each PR only once. 
It is also good practice to do that on the master-branch only, and not in the PR itself. Otherwise we might run into problems if two PRs are open at the same time, as they will use the same new Version. Thus, the PR does not increment, but master increments, after the PR has been merged,

Reply:
I changed this, by resolving the merge conflict this change caused. You don't have to change anything here anymore.